### DISCUSSION TITLE
Mapping new supplier in new leads

### DISCUSSION THREAD

**Discussion by gsportelli73** [YetiForceCompany/YetiForceCRM#14249] [How to]
Hello, 
i have a question, i have created a map between supplier and lead.
I want to create new leads when i create new supplier into CRM.
I have create the mapping match. Watch video 
[https://youtu.be/IV2nQ9q55Rg](url)

Now i think that is correct.
But it doesn't function.
For me best solution was create a workflow that when i create new supplier, it automatically creates new lead. 
This is important for passive cycle of my company, because sometime it must order a product or service for it.
Thank for attention

**Comment by apcloic** [comment]
@gsportelli73
Hi, 
Check if all required fields are mapped, including "assigned to" field otherwise it won't work. 

For this kind of issue, plz post your question in discussions section. 
#13908

Regards,

**Reply by gsportelli73** [reply]
Hello, thank for answer 
I doesn't resolve.
Watch video 
[]https://youtu.be/3zaQcFqIjBc(url)

Infact when i create a new supplier no lead created.
Now i don't think that workflow on new lead that i have setted when a web form on wordpress submitted can it create problem ? is it correct ?
Thank you
